docs/_includes/attr-doc.adoc

```diff
@@ -112,6 +126,65 @@ Another approach is to change the order of substitutions that are applied where
 The application is called {app-name}.
 ----
 
+==== Splitting attribute entry values over multiple lines
+
+===== Splitting long value strings
+
+Use backslashes `\` at the end of each attribute value to split long listings.
```

Comment:
I'd rather it be:

"Use a single space followed by a backslash at the end of each line... The space will be preserved in the final value."

I'm not sure we want to break into an admonition here because it seems excessive and breaks the flow. I think simply stating that the space is required, and that it is preserved, allows the reader to conclude that this can only be used where there is a natural break in the text.